### PR TITLE
[CALCITE-3204] Implement jps command for OS adapter

### PR DESCRIPTION
**Pull request by XuQianJin-Stars**
fix issue [CALCITE-3204](https://issues.apache.org/jira/browse/CALCITE-3204)

### CODE REVIEW

File: plus/src/main/java/org/apache/calcite/adapter/os/JpsTableFunction.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.type.SqlTypeName;
+import org.apache.calcite.util.ImmutableBitSet;
+
+import com.google.common.collect.ImmutableList;
+
+/**
+ * Table function that executes the OS "jps" ("Java Virtual Machine Process Status Tool") command
+ * to provides a display of all current java process pids.
```
Comment (hsyuan) [inline plus/src/main/java/org/apache/calcite/adapter/os/JpsTableFunction.java]: to list all java processes of a user.

**Comment by zinking** [comment]
how about also add the jvm args column ?

**Comment by XuQianJin-Stars** [comment]
The CI is broken.

**Comment by hsyuan** [comment]
@zinking Can you give an example?

**Comment by julianhyde** [comment]
I am surprised that `jps` does not work on Windows. It should work on any platform with a JRE, right? It's not like `du` and `ps` whose tests are rightly disabled because they are Unix commands.

**Comment by XuQianJin-Stars** [comment]
> I am surprised that `jps` does not work on Windows. It should work on any platform with a JRE, right? It's not like `du` and `ps` whose tests are rightly disabled because they are Unix commands.

hi @julianhyde thanks,  well, I have modified it.

**Comment by XuQianJin-Stars** [comment]
hi @hsyuan @julianhyde Thank you very much, I have addressed.

**Pull request closed without merging**